app/models/offsite_payment.rb

```diff
@@ -6,5 +6,6 @@ class OffsitePayment < ActiveRecord::Base
 	belongs_to :donation
 	belongs_to :nonprofit
 	belongs_to :supporter
+	belongs_to :offline_transaction_charge
```

Comment:
Based upon the suggested change in `app/models/offline_transaction_charge.rb`, this should be a has_one that goes through payment

Reply:
This should be handled verified in `spec/models/offsite_payment_spec.rb`. Which doesn't actually exist yet apparently but should.

Reply:
Are u sure we can set up has_one offline transaction charge to offsite payment? I don't think offline_transaction_charge is connected to payment.

Reply:
This can be done but we have to handle is slightly differently because we have to specify the particular fields on `Payment` to join through.

Do we have a use-case where we know we need to get the `offline_transaction_charge` from an `offsite_payment`? If we don't immediately, it might just be easiest to remove the reference inside `OffsitePayment` and not worry about it.

Reply:
I don't know of a use case like that - I support just removing the reference.

Reply:
@wwahammy Done, lmk what you think